llvm/lib/Target/AIE/AIE2InstrInfo.td

```diff
@@ -584,8 +585,8 @@ class Split2DInstr<Instruction RealInst, int opidx> : SplitPseudo<RealInst,
 foreach instr = [VLDA_2D_dmw_lda_w, VLDA_2D_dmw_lda_am, VLDA_2D_CONV_FP32_BF16,
                  VLDB_2D, VLDB_2D_128, LDA_2D_dmv_lda_q, VLDB_2D_UNPACK_S8_S4,
                  VLDB_2D_UNPACK_S16_S8, VLDB_2D_UNPACK_D8_D4, VLDB_2D_UNPACK_D16_D8,
-                 VST_2D_dmw_sts_w, VST_2D_dmw_sts_am, VST_2D_128, ST_2D_dmv_sts_q,
-                 LDA_2D_dms_lda, LDA_2D_S8_dmhb_lda, LDA_2D_U8_dmhb_lda,
+                 VLD_2D_pseudo, VST_2D_dmw_sts_w, VST_2D_dmw_sts_am, VST_2D_128,
```

Comment:
Note : This is critical for the `aie-superreg-rewrite` to work as expected. This helps in avoiding spills of the `D` type regs.